### DISCUSSION TITLE
⛔ Set specific sections printable with Share Button Widget

### DISCUSSION THREAD

**Discussion by incircolo** [elementor/elementor#14102] [Feature Request]
Hi to all
It's not clear to me if it's possibile to do this or if it's needed something as a new feature.
The problem is that when i click to "print" button under Share Buttons Widget i get the entire page content and also the footer section under it, but i'd like to choose only specific sections to print or at least exclude header and footer templates.
Nice thing should be to have the possibility to "tag" the printable sections with some custom id or CSS class (ie. adding "print" in css class section make it printable).
Thanks

**Comment by nicholaszein** [comment]
Hello @incircolo,

Thank you for submitting a feature request! 🙏 

### ⛔ 🚀 After reviewing your request, we have decided not to add this to our roadmap.

What you are requesting can be achieved by using Custom CSS with the `@media print` media query.

Please feel free to contact us again with other feature suggestions. 😉 

Best regards
